libs/crossplatform/include/CrossPlatformUtils.h

```diff
@@ -69,6 +69,13 @@ class CrossPlatformUtils
   */
   static const std::string& GetHostType();
 
+  /**
+   * @brief Checks if the given host type matches the OS this code is built for: Linux, Windows or Mac OS
+   * @param host type given as "win", "linux", "mac", "all" or "" (the last two always match)
+   * @return true if the given host type matches the host platform
+  */
+  static bool MatchesHost(const std::string& hostType);
```

Comment:
The method does not belong to cross-platform library. The constants "win", "linux", "mac", "all" are specific to CMSIS standard and should only be used in that context.

Reply:
The string taken as parameter by `CrossPlatformUtils::MatchesHost()` exactly matches what is returned by `CrossPlatformUtils::GetHostType()`. I don't see how that is different. However, since this function isn't used anymore (since I removed `RteGenerator::GetArguments()`), I could remove it.

Reply:
It kind of fulfills the same purpose as `RteGenerator::GetExpandedCommandLine()`. That function didn't fulfill my needs, so I wanted the same information but in a more usable form than a text blob.

Reply:
I agree.

Reply:
Should be removed now.